content/updates/2018-06-25-release-notes.md

```diff
@@ -0,0 +1,28 @@
++++ 
+date = "2018-06-25" 
+title = "cloud.gov update: Buildpack notifications, better backup protection, and more" 
++++ 
+
+It's been a few months since our last update, so there’s more to see here than usual. As always, get in touch with us if you have any questions or feedback about these features. 
+
+### Announcements
+
+* We’ve switched dedicated RDS instances from 24 hours to 14 days backup retention. Please note that for shared plans (shared-psql and shared-mysql), [RDS does not back up your data]({{< relref "docs/services/relational-database.md" >}}).
+
+### Added
+
+* Since AWS CloudFront is outside the AWS FedRAMP P-ATO boundary, we offer [a custom domain service without CloudFront]({{< relref "/updates/2018-05-21-new-custom-domain-service.md" >}}).  If you’re using the CDN service and you need to switch to the custom domain service, follow the instructions on the [custom domain page]({{< relref "/docs/services/custom-domains.md" >}}).
+
+* Support for SpringBoot “thin” JARs has been added to the Java buildpack in CloudFoundry. 
+
+### Buildpacks 
+
```

Comment:
Can we make this section a bulleted list?

Here’s an example. Should make it more legible.
```
- item one
- item two
```

- item one
- item two